.github/workflows/ci.yml

```diff
@@ -17,8 +17,20 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
+        pip install --upgrade pip setuptools wheel
         pip install --upgrade -r requirements.txt
-        pip install .
+    - name: Build package
+      run: |
+        python ./setup.py bdist_wheel
+        mv ./dist/*.whl ./dist/netaddr-0.CI-py2.py3-none-any.whl
+    - name: Upload the wheel artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheel
+        path: ./dist/netaddr-0.CI-py2.py3-none-any.whl
+    - name: Install package
+      run: |
+        pip install ./dist/netaddr-0.CI-py2.py3-none-any.whl
```

Comment:
Is any of the changes in this file necessary? It seems to me that the answer is no and those are out of scope but I may be missing something.

Reply:
* `setuptools` available by default in the env is damn old (22) and seems to be unable to update itself even if specified (for setup.cfg 30.3.0 (dated to December 2016) is required).
* explicit building wheel is needed for `setuptools_scm`-introducing PR - while the code is installed, it is still imported from the relative dir. `setuptools_scm` generates a file with version, but when `pip install .`, that file goes to a temp dir and then into wheel. When building explicitly the file goes to the repative dir.